nestkernel/nest_time.h

```diff
@@ -387,7 +387,13 @@ class Time
   bool
   is_neg_inf() const
   {
-    return tics == LIM_NEG_INF.tics;
+    return tics <= LIM_NEG_INF.tics;
```

Comment:
@heplesser so, the choices are: leave as is, replace the inequality portion with an assertion, remove the inequality.